Simulations/grids.yaml

```diff
@@ -0,0 +1,20 @@
+plugins:
+  source:
+    - module: intake_xarray
+
+sources:
+
+  EUREC4A_PR1250m_DOM01:
+    args:
+      consolidated: true
+      urlpath: https://swift.dkrz.de/v1/dkrz_948e7d4bbfbb445fbff5315fc433e36a/grids/EUREC4A_PR1250m_DOM01.zarr
+    description: ICON grid of DOM01
+    driver: zarr
+
+  EUREC4A_PR1250m_DOM02:
+    args:
+      consolidated: true
+      urlpath: https://swift.dkrz.de/v1/dkrz_948e7d4bbfbb445fbff5315fc433e36a/grids/EUREC4A_PR1250m_DOM02.zarr
+    description: ICON grid of DOM02
+    driver: zarr
+
```

Comment:
I'd suggest to use the grid UUIDs as keys in this catalog. (Probably also just as an addition to the current entries). That would allow for a usage pattern similar to what has been shown in [Getting started with Monsoon 2.0]():
```python
data = cat.simulations.ICON.experiment_1.surface_DOM01.to_dask()
grid = cat.simulations.grids[data.uuidOfHGrid].to_dask()
dwg = data.merge(grid)
```
an advantage of this approach might be, that it can be easier to swap out the data and the grid will adjust iteslf automatically.

I also was wondering where to find the vertical grids which are referenced by some datasets.

Reply:
Done with 2ff37bcc375be97d10dde09392de0ad59460d444. Vertical grids are not yet available and indexes. I'll keep that in mind for a later PR.